src/DensityTest.jl

```diff
@@ -0,0 +1,272 @@
+# Density test for manipulation in the running variable. 
+# This work is based on McCrary, J. (2008). Manipulation of the running variable in the regression discontinuity design: A density test. Journal of econometrics, 142(2), 698-714.
```

Comment:
Hi Pablo! Can you maybe rename the file to `density_test.jl` so that it matches the other file names in `src`?

Reply:
Hi Nikos! I will be going answering to each comment one at a time. I renamed the file and the functions according to the naming convention. I'm new at this so all the comments are appreciated!